common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -342,7 +414,7 @@ public void run() {
                     Logger.info(TAG + methodName,
                             "Completed duplicate request with correlation id : **"
                                     + command.getParameters().getCorrelationId() + ", having the same result as : "
-                                    + result.getCorrelationId() + ", with the status : "
+                                    + result.getCorrelationId() + statusMsg()
                                     + result.getStatus().getLogStatus());
```

Comment:
You might want to just pass the status over to your helper method and just construct the complete String there